benchmark/scripts/Benchmark_Driver

```diff
@@ -627,16 +627,26 @@ class BenchmarkDoctor(object):
     @staticmethod
     def _constant_memory_use(measurements):
         select = BenchmarkDoctor._select
+        name = measurements["name"]
+
+        memory_uses = [
+            [r.mem_pages for r in i_series if r.mem_pages is not None]
+            for i_series in [select(measurements, num_iters=i) for i in [1, 2]]
+        ]
+        memory_uses = [m for m in memory_uses if m]
```

Comment:
@tbkka could you please take a look at this part of the PR? This fixes a `TypeError` that was thrown out of this Python code when `memory_uses` is either empty or only contains empty elements.

cc @shahmishal

Reply:
Looks fine.